std/range/package.d

```diff
@@ -6592,13 +6592,12 @@ Transversal!(RangeOfRanges, opt) transversal
     assert(mat1[1] == 10);
 }
 
-struct Transposed(RangeOfRanges)
+struct Transposed(RangeOfRanges,
+    TransverseOptions opt = TransverseOptions.assumeJagged)
```

Comment:
Now you've done it :) We need to implement the assumption for all of transposed if we are going this route. Which gives us a lot of good options, nice idea!

Reply:
BTW, I didn't even know about `transverse`, this is a perfect solution for it!